web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/PageComponents/AppTopBar/Environment/index.tsx

```diff
@@ -20,7 +20,7 @@ export const Environment = (props: EnvironmentProps) => {
             <StartUpIcon className="w-4 h-auto text-system-success-300 " />
           )}
           {environment === "staging" && (
-            <SmartPhoneIcon className="w-4 h-auto text-system-warning-300" />
+            <SmartPhoneIcon className="w-4 h-auto text-system-warning-500" />
           )}
```

Comment:
I am making a personal decision that this looks better since the other yellow is too light